### PR TITLE
New address bar

### PR DESCRIPTION
**Pull request by howard-stearns**
Implement new address bar wireframe, forward compatible with 'announcement' type stories.

This restores the original horizontal design, but within the tablet, and with separate dedicated rows for places vs snaps.

Actual detailed design is still to come, but that will be a separate PR. I want people to have experience with this while the detailed design is going on.

### CODE REVIEW

**Comment by howard-stearns** [comment]
Test plan:
I will revise test plan library for suggestions in the obvious way but you don't need to wait for that. Obviously: no blue chevron and no All/Places/Snaps tabs.

Also, if you edit resources/qml/hifi/tablet/TabletAddressDialog.qml, and look for the line that says 'uncomment this line', you can get simulated 'announcement' data to see what that will look like.

**Comment by hifi-gustavo** [comment]
Build console output for commit: e13237dade0c644ff0c267cbfefc26223c543328 available at: https://gist.github.com/37114b58381addd7b497f5cbb52bda5d<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10253-e13237d.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10253-e13237d.dmg

**Comment by howard-stearns** [comment]
Note: we're working  on UI/UX polish for this, which will come in a second PR. This is just about functionality.  Some of the expected changes to come are:
- Different sizes, such that the vertical scroll won't be necessary.
- Different colors and "whitestrip" behavior on the cards.

File: interface/resources/qml/hifi/Card.qml [hunk 1]
```diff
@@ -129,7 +130,7 @@ Rectangle {
     property int dropSamples: 9;
     property int dropSpread: 0;
     DropShadow {
-        visible: true;
+        visible: showPlace && desktop.gradientsSupported;
```
Comment (zfox23) [inline interface/resources/qml/hifi/Card.qml]: Does the reference to `desktop` cause problems in the tablet?

File: interface/resources/qml/hifi/Card.qml [hunk 2]
```diff
@@ -218,37 +206,22 @@ Rectangle {
...
-        id: rectIcon
-        z: 1
-        width: 32
-        height: 32
-        radius: 15
+    StateImage {
+        id: actionIcon;
+        imageURL: "../../images/info-icon-2-state.svg";
+        size: 32;
+        buttonState: messageArea.containsMouse ? 1 : 0;
```
Comment (zfox23) [inline interface/resources/qml/hifi/Card.qml]: `containsMouse` is way slower than using `onEntered` and `onExited` within the `messageArea` MouseArea itself. I'd prefer this behavior to be moved down there.

File: interface/resources/qml/hifi/Feed.qml [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+        highlightMoveVelocity: -1;
+        highlight: Rectangle { color: "transparent"; border.width: 4; border.color: hifiStyleConstants.colors.blueHighlight; z: 1; }
+
+        spacing: 14;
+        width: parent.width;
+        height: cardHeight + stackedCardShadowHeight;
+        delegate: Card {
+            width: cardWidth;
+            height: cardHeight;
+            goFunction: goCard; // fixme global
```
Comment (zfox23) [inline interface/resources/qml/hifi/Feed.qml]: fixme for the future or for now?

File: interface/resources/qml/hifi/Card.qml (hunk 1 above)
Comment (howard-stearns) [inline interface/resources/qml/hifi/Card.qml]: i'll check. thanks.

File: interface/resources/qml/hifi/Card.qml (hunk 2 above)
Comment (howard-stearns) [inline interface/resources/qml/hifi/Card.qml]: I'll check it out. thanks.

File: interface/resources/qml/hifi/Feed.qml (hunk 3 above)
Comment (howard-stearns) [inline interface/resources/qml/hifi/Feed.qml]: I had meant it for now. I'll see if I can address it now.

File: interface/resources/qml/hifi/Card.qml (hunk 2 above)
Comment (howard-stearns) [inline interface/resources/qml/hifi/Card.qml]: I'll save this for the ux round.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 56196c03a355beb7a32ea006de1b6ebbca789c06 available at: https://gist.github.com/37114b58381addd7b497f5cbb52bda5d<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10253-56196c0.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10253-56196c0.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: e4f259e6303dd64da07b47631fd0a6a3ab8237ec available at: https://gist.github.com/37114b58381addd7b497f5cbb52bda5d<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10253-e4f259e.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10253-e4f259e.dmg

**Comment by howard-stearns** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: e4f259e6303dd64da07b47631fd0a6a3ab8237ec available at: https://gist.github.com/37114b58381addd7b497f5cbb52bda5d<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10253-e4f259e.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10253-e4f259e.dmg

**Pull request merged**